label_studio_ml/examples/bert/bert_classifier.py

```diff
@@ -64,15 +64,15 @@ def __init__(
             ))
 
     def reset_model(self, pretrained_model, cache_dir, device):
-        model = BertForSequenceClassification.from_pretrained(
+        self.model = BertForSequenceClassification.from_pretrained(
             pretrained_model,
             num_labels=len(self.labels),
             output_attentions=False,
             output_hidden_states=False,
             cache_dir=cache_dir
         )
-        model.to(device)
-        return model
+        self.model.to(device)
```

Comment:
reset_model is used during training.
It's better to fix this in __init__ adding:
`self.model = self.reset_model`

Reply:
Ah apologies, yes of course you're right